### PR TITLE
chore: Update to use flutter_gpu package following launch [flame_3d]

### PR DESCRIPTION
**Pull request by luanpotter**
# Description

Update to use flutter_gpu package following launch.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org/
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -27,18 +25,7 @@ jobs:
...
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: ${{env.FLUTTER_MIN_VERSION}}
-      - uses: bluefireteam/melos-action@v3
-      - name: "Analyze with lowest supported version"
-        uses: invertase/github-action-dart-analyzer@v2.0.0
-        with:
-          fatal-infos: true
-
-  analyze-latest:
```
Comment (luanpotter) [inline .github/workflows/cicd.yml:37]: this will only work on main so no reason to have two analyzes

Comment (spydon) [reply .github/workflows/cicd.yml:37]: Maybe better to set it to `if: false`, so that we don't accidentally remove it on a merge later?

**Review by spydon — APPROVED** [review]

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (spydon) [reply .github/workflows/cicd.yml:37]: Or maybe that would actually be even more prone to slip through :sweat_smile:

Comment (luanpotter) [reply .github/workflows/cicd.yml:37]: up to you! haha either way wroks for me

Comment (spydon) [reply .github/workflows/cicd.yml:37]: Not changing anything in the PR is easier, path of least resistance 😄

**Pull request merged**